kivy/app.py

```diff
@@ -654,7 +661,10 @@ def get_application_config(self):
         '''
 
         if platform == 'android':
-            defaultpath = '/sdcard/.%(appname)s.ini'
+            defaultpath = join(
+                dirname(abspath(sys.executable)),
```

Comment:
works just fine with Pyonic on python 2 :) the env var is set in `PythonActivity.java` + `start.c`, so we actually just need to make sure we don't change the core that launches the app (and if we change, it won't launch), so we should be quite safe with this one, because it'd be probably the first thing to fix.